aimmo-game/tests/test_service.py

```diff
@@ -38,67 +34,92 @@ def setup_world(self):
+
+    def test_healthy_flask(self):
+        """
+        Tests the flask service. HEALTHY is returned if the app can be routed to root.
+        """
+        service.app.config['TESTING'] = True
+        self.app = service.app.test_client()
+        response = self.app.get('/')
+        self.assertEqual(response.data, 'HEALTHY')
+
+    def test_player_dictionary(self):
+        """
+        Ensures the "players" element of the get_world_state() JSON returns the correct information for the dummy
+        avatar provided into the world.
+
+        NOTE: Orientation (and others) may be hard coded. This test WILL and SHOULD fail if the functionality is added.
+        """
+        player_list = self.world_state_json['players']
+        self.assertEqual(len(player_list), 1)
+        details = player_list[0]
         self.assertEqual(details['id'], 1)
-        self.assertEqual(details['x'], 0)
-        self.assertEqual(details['y'], -1)
+        self.assertEqual(details['location']['x'], 0)
+        self.assertEqual(details['location']['y'], -1)
         self.assertEqual(details['health'], 5)
+        self.assertEqual(details['orientation'], "north")
         self.assertEqual(details['score'], 0)
 
     def test_score_locations(self):
-        result = self.setup_world()
-        self.assertEqual(result['score_locations'], [(0, 1)])
-
-    def test_width(self):
-        result = self.setup_world()
-        self.assertEqual(result['width'], 2)
-
-    def test_height(self):
-        result = self.setup_world()
-        self.assertEqual(result['height'], 3)
-
-    def test_layout(self):
-        result = self.setup_world()
-        expected = {
-            0: {
-                -1: 0,
-                0: 0,
-                1: 2,
-            },
-            1: {
-                -1: 0,
-                0: 1,
-                1: 0,
-            }
-        }
-        self.assertEqual(result['layout'], expected)
-
-    def test_min_x(self):
-        result = self.setup_world()
-        self.assertEqual(result['minX'], 0)
-
-    def test_min_y(self):
-        result = self.setup_world()
-        self.assertEqual(result['minY'], -1)
-
-    def test_max_x(self):
-        result = self.setup_world()
-        self.assertEqual(result['maxX'], 1)
-
-    def test_max_y(self):
-        result = self.setup_world()
-        self.assertEqual(result['maxY'], 1)
-
-    def test_pickup_list(self):
-        result = self.setup_world()
-        self.assertIn({'name': 'a', 'location': (1, 1)}, result['pickups'])
-        self.assertIn({'name': 'b', 'location': (0, -1)}, result['pickups'])
+        """
+        Ensures the correct score location in the "score_locations" element; is returned by the JSON.
+        """
+        score_list = self.world_state_json['score_locations']
+        self.assertEqual(score_list[0]['location']['x'], 0)
+        self.assertEqual(score_list[0]['location']['y'], 1)
+
+    def test_north_east_corner(self):
```

Comment:
Nit: Shouldn't these test be for the WorldMap class?

Reply:
They don't necessarily test the map. They test that the correct JSON is outputted. I'll change the name.